csrc/kernel_cache.cpp

```diff
@@ -975,14 +975,11 @@ FusionKernelRuntime::FusionKernelRuntime(
     fusion->printMath();
   }
 
-  all_tvs_ = ir_utils::allTvs(fusion.get());
-
-  // Run segmentation on the copied fusion
+  // SchedulerRuntimeInfo modifies the fusion, so it is required for both
+  // compile paths.
+  std::vector<TensorView*> all_tvs = ir_utils::allTvs(fusion.get());
   SchedulerRuntimeInfo runtime_info(
-      fusion.get(), args, nullptr, all_tvs_, forced_index_type);
-
-  // Initialize the evaluator simplifer
-  precomputed_values_ = std::make_unique<PrecomputedValues>(fusion.get());
```

Comment:
QQ: I'm wondering what are we losing here from now caching `precomputed_values_` for each segments.

Reply:
We probably have to recompute values common to each segmented fusion.

Reply:
> We probably have to recompute values common to each segmented fusion.

And [that](https://github.com/NVIDIA/Fuser/pull/1649/files/08cffeea3446f760a47cbfae59cf02bd100036df#diff-69db587a93b7b0fefd57640c68f0f1faaca8ece64173d757d17d8962e502ecd6R1639) is on the critical path, correct? Do you see an increase on the critical path latency with this change?